src/main/io/serial_cli.c

```diff
@@ -588,29 +588,42 @@ static void cliRxFail(char *cmdline)
 
             uint16_t value;
             rxFailsafeChannelMode_e mode = channelFailsafeConfiguration->mode;
-            bool requireValue = channelFailsafeConfiguration->mode == RX_FAILSAFE_MODE_SET;
+            bool requireValue = ((mode == RX_FAILSAFE_MODE_AUTO_OR_SET) && (channel >= NON_AUX_CHANNEL_COUNT));
```

Comment:
You mean to use in the code actually below that line ?

Like this:
~~~C
            bool isFlightChannel = channel < NON_AUX_CHANNEL_COUNT;

            // TODO optimize to use rxFailsafeModes - less logic.
            ptr = strchr(ptr, ' ');
            if (ptr) {
                switch (*(++ptr)) {
                    case 'h':
                        mode = RX_FAILSAFE_MODE_HOLD;
                        parseError = false;
                        break;

                    case 'a':
                        if (isFlightChannel) {
                            mode = RX_FAILSAFE_MODE_AUTO_OR_SET;
                            parseError = false;
                        }
                        break;

                    case 's':
                        if (!isFlightChannel) {
                            mode = RX_FAILSAFE_MODE_AUTO_OR_SET;
                            parseError = false;
                        }
                        break;

                    default:
                        break;
                }
~~~